package.json

```diff
@@ -39,6 +39,7 @@
     "@parcel/compressor-brotli": "2.8.2",
     "@parcel/compressor-gzip": "2.8.2",
     "@parcel/packager-raw-url": "2.8.2",
+    "@parcel/transformer-typescript-tsc": "^2.8.2",
```

Comment:
Any idea why this dependency has been introduced?

Reply:
It was a left over from old changes. Initially I had added parcel-bundle-report but switched to --detailed-reports which is more integrated with the build.